example_sdk_client/example_sdk_client_simulator.py

```diff
@@ -39,15 +40,22 @@ def handle_on_progress_update(job: Job, progress_update: JobProgressUpdate):
 
     omotes_if.submit_job(
         esdl=base64.b64decode(input_esdl).decode(),
-        params_dict={"key1": "value1", "key2": ["just", "a", "list", "with", "an", "integer", 3]},
-        workflow_type=WorkflowType("simulator", "some descr"),
+        params_dict={
```

Comment:
It would be nicer to have `time_step_s`, `time_start_unix_s` and `time_end_unix_s` for uniformity and alphabetical sorting.
But maybe too much hassle to change due to dependencies.